### DISCUSSION TITLE
Slides do Curso de Fundamentos.

### DISCUSSION THREAD

**Discussion by fernandocfabril** [balta-io/forum#557] [Tira Dúvidas] [answered]
Boa tarde.
Eu não encontrei os Slides que foram apresetandos no curso Fundamentos do C#.
Poderia, por favor, me mostrar onde eu posso acessar eles?

Obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @fernandocfabril, como vai?
Segue: https://github.com/balta-io/2801/tree/master

**Reply by fernandocfabril** [reply]
Bom dia, @andrebaltieri, muito obrigado.
